### PR TITLE
[merged] Remove unused and bogus ;

### PR DESCRIPTION
**Pull request by rhatdan**
Beginning to cleanup code to remove warnings from pylint

Testing with

	$(PYTHON3_PYLINT) --disable=all --enable=E --enable=W --additional-builtins=_ *.py atomic Atomic tests/unit/*.py

Reveals these unused and bogus ;.

### CODE REVIEW

**Comment by cgwalters** [comment]
There's a bit more here than removal of `;`, but sure, looks sane to me.

@rh-atomic-bot r+ 048988b

**Comment by rhatdan** [comment]
@cgwalters I will have some more coming.

File: Atomic/util.py [hunk 1]
```diff
@@ -241,9 +241,9 @@ def skopeo_layers(image, args=[], layers=[]):
             raise ValueError("Unable to interact with this registry: {}".format(r.stderr))
     except OSError:
         raise ValueError("skopeo must be installed to perform remote inspections")
-    except Exception as e:
+    finally:
```
Comment (giuseppe) [inline Atomic/util.py]: this changes the logic of the function and rm the directory even on success.  We should rm the temporary directory only on an exception.  I'll post another patch to address it.

Comment (giuseppe) [inline Atomic/util.py]: PR here: https://github.com/projectatomic/atomic/pull/464

**Pull request closed without merging**